packages/react-native-bridge/index.js

```diff
@@ -259,4 +259,22 @@ export function requestMediaFilesEditorLoad( mediaFiles, blockClientId ) {
 	);
 }
 
+export function requestMediaFilesFailedRetryDialog( mediaFiles ) {
+	return RNReactNativeGutenbergBridge.requestMediaFilesFailedRetryDialog(
+		mediaFiles
+	);
+}
```

Comment:
Could we add JDoc inline documentation? Check `requestMediaPicker` as an example.
I believe the `return` is not needed 🤔

Also, would it make sense to have only one method with a flag describing the kind of dialog requested?

Reply:
> Could we add JDoc inline documentation? Check requestMediaPicker as an example.
I believe the return is not needed 🤔

👍 done in bc2f577c2e (added docs) and (removed unneeded return statements) in 0817df1b75


> Also, would it make sense to have only one method with a flag describing the kind of dialog requested?

I see - do you mean like an enum? I was following along the lines of media upload retry and cancel dialog signaling methods as seen here:

```
export function requestImageFailedRetryDialog( mediaId ) {
	return RNReactNativeGutenbergBridge.requestImageFailedRetryDialog(
		mediaId
	);
}

export function requestImageUploadCancelDialog( mediaId ) {
	return RNReactNativeGutenbergBridge.requestImageUploadCancelDialog(
		mediaId
	);
}
```

It made sense to keep things consistent this way so to reflect the "API design" if anything. I think it may demand a slightly bigger refactor separated from the Stories functionality to make maybe one bridge method and consist all these 5 signals (3 for mediaFiles collection based signals and the remaining preexisting 2 for single media items) there for requesting the host app to show one of the 5 variants of dialogs. Wdyt?

Reply:
> I think it may demand a slightly bigger refactor separated from the Stories functionality to make maybe one bridge method and consist all these 5 signals 

Yes I think you are right! 👍